internal/config/mount_config.go

```diff
@@ -106,9 +110,13 @@ type FileSystemConfig struct {
 }
 
 type FileCacheConfig struct {
-	MaxSizeMB             int64 `yaml:"max-size-mb"`
-	CacheFileForRangeRead bool  `yaml:"cache-file-for-range-read"`
-	EnableCrcCheck        bool  `yaml:"enable-crc-check"`
+	MaxSizeMB                  int64 `yaml:"max-size-mb"`
+	CacheFileForRangeRead      bool  `yaml:"cache-file-for-range-read"`
+	EnableParallelDownloads    bool  `yaml:"enable-parallel-downloads"`
+	DownloadParallelismPerFile uint  `yaml:"download-parallelism-per-file"`
```

Comment:
Use unsigned integers sparingly: refer: https://yaqs.corp.google.com/eng/q/8035320036228857856. Same for ReadRequestSizeMB.